docs/integration.rst

```diff
@@ -1241,6 +1243,124 @@ argument in the ``efibootmgr`` call above:
   type=ext4
   bootname=system1
 
+Raspberry Pi firmware
+~~~~~~~~~~~~~~~~~~~~~
+
+The Raspberry Pi firmware supports fail-safe OS updates with A/B booting thanks
```

Comment:
I have doubts about the power-fail safety of this scheme given the use of FAT. Perhaps we should have a warning that suggests barebox-state or U-Boot redundant environment placed outside a file system instead of this until concerns are resolved.

Reply:
I have added such warning in 1f67e0112c417d1e9a9bdb9d3a8c725153c492f8.